tests/test_costs.py

```diff
@@ -31,6 +31,22 @@ def test_filters(self):
             assert len(results) == 1
             assert results[0][column.key] == values[0]
 
+    def test_pagination_offset(self):
+        factories.CommunicationCostFactory()
+        response = self._response(api.url_for(CommunicationCostViewOffset))
+        print('response is: ', response, '\n')
```

Comment:
should comment out "print", it is fine for this 30 days code.

Reply:
Thanks @fec-jli ! I removed that line 👍